cmd/control-router.go

```diff
@@ -79,7 +79,7 @@ type controlAPIHandlers struct {
 }
 
 // Register control RPC handlers.
-func registerControlRPCRouter(mux *router.Router, srvCmdConfig serverCmdConfig) {
+func registerControlRPCRouter(mux *router.Router, srvCmdConfig serverCmdConfig) (err error) {
```

Comment:
Do you really need a named return here? You do not `defer` nor use naked returns